Tests/Unit/DependencyInjection/ConfigurationTest.php

```diff
@@ -119,6 +119,7 @@ public function testSupportsAllConfigFormats()
                     'factory' => 'httplug.factory.guzzle6',
                     'http_methods_client' => true,
                     'service' => null,
+                    'public' => false,
```

Comment:
this should now be `null`, right?

Reply:
Changed it... Somehow it didn't fail when it was either `null` or `false`.